cmd/stratus/main.go

```diff
@@ -9,6 +11,10 @@ var rootCmd = &cobra.Command{
 	Use: "stratus",
 }
 
+var (
+	workerCount = runtime.GOMAXPROCS(0)
```

Comment:
Since the bottleneck is the AWS API / network, would it make sense to have as many workers as TTPs to warmup/detonate/cleanup?

It would make sense to restrict to GOMAXPROCS if the task was CPU-bound, but that's not the case

Reply:
implemented with 1 worker = 1 attack technique  572ea44, let me know if you have a strong opinion against it